validation/immersed_boundaries/flow_around_cylinder.jl

```diff
@@ -0,0 +1,118 @@
+# # Steady-state flow around a cylinder in 2D using immersed boundaries
+
+using Statistics
+using Plots
+using JLD2
+using Printf
+
+using Oceananigans
+using Oceananigans.Advection
+using Oceananigans.Grids
+using Oceananigans.Fields
+using Oceananigans.AbstractOperations
+using Oceananigans.OutputWriters
+
+# setting boundary condition topology
+topology=(Periodic, Bounded, Bounded)
+
+# setting up 2D grid
+grid = RegularCartesianGrid(topology=topology, size=(350, 350, 1), x=(20, 40), y=(10, 30), z=(0, 1))
+#grid = RegularCartesianGrid(topology=topology, size=(1500, 1500, 1), x=(0, 60), y=(0, 60), z=(0, 1))
+
+
+# reynolds number
+Re = 40
+
+#cylinder with center at (30,20)
+const R = 1 # radius
+inside_cylinder(x, y, z) = ((x-30)^2 + (y-20)^2) <= R # immersed solid
+
+# boundary conditions: inflow and outflow in y
+v_bcs = VVelocityBoundaryConditions(grid,
+                                    north = BoundaryCondition(NormalFlow,1.0),
+                                    south = BoundaryCondition(NormalFlow,1.0))
+
+# setting up incompressible model with immersed boundary
+model = IncompressibleModel(timestepper = :RungeKutta3, 
+                              advection = UpwindBiasedFifthOrder(),
+                                   grid = grid,
+                               buoyancy = nothing,
+                                tracers = nothing,
+                                closure = IsotropicDiffusivity(ν=1/Re),
+                    boundary_conditions = (v=v_bcs,),
+                      immersed_boundary = inside_cylinder
+                           )
+
+# initial condition
+# setting velocitiy to zero inside the cylinder and 1 everywhere else
+v₀(x, y, z) = ifelse(inside_cylinder(x,y,z),0.,1.)
+set!(model, v=v₀)
+
+
+progress(sim) = @info @sprintf("Iteration: % 4d, time: %.2f, max(v): %.5f, min(v): %.5f",
+sim.model.clock.iteration,
+sim.model.clock.time,
+maximum(sim.model.velocities.v.data),
+minimum(sim.model.velocities.v.data))
+
+simulation = Simulation(model, Δt=5.7e-3, stop_time=5, iteration_interval=10, progress=progress)
+
+# ## Output
+
+simulation.output_writers[:fields] = JLD2OutputWriter(model,
+                                                      merge(model.velocities, model.pressures),
+                                                      schedule = TimeInterval(0.5),
+                                                      prefix = "flow_around_cylinder",
+                                                      force = true)
+
+# run it
+run!(simulation)
+
+# Analyze Results
+file = jldopen(simulation.output_writers[:fields].filepath)
+
+iterations = parse.(Int, keys(file["timeseries/t"]))
+
+u, v, w = model.velocities
+
+xv, yv, zv = nodes(v)
+
+# defining a function to mark boundary
+function circleShape(h,k,r)
```

Comment:
```suggestion
function circle_shape(h, k, r)
```

We follow the convention that objects are `TitleCase` and functions are `snake_case`, and always put spaces after commas!

Reply:
Ah thanks for letting me know! I'll try to catch those next time.